tests/manual-test-cases/Group5-Functional-Tests/5-22-NFS-Volume.md

```diff
@@ -0,0 +1,83 @@
+Test 5-22 - NFS Volume
+=======
+
+# Purpose:
+To verify that NFS shared volumes work with currently supported docker commands
+
+# References:
+[1 - Best practices for running VMware vSphere on NFS](http://www.vmware.com/content/dam/digitalmarketing/vmware/en/pdf/techpaper/vmware-nfs-bestpractices-white-paper-en.pdf)
+
+[2 - Docker Command Line Reference - Volume Create](https://docs.docker.com/engine/reference/commandline/volume_create/)
+
+[3 - Docker Command Line Reference - Exec](https://docs.docker.com/engine/reference/commandline/exec/)
+
+[4 - Docker Command Line Reference - Volume Inspect](https://docs.docker.com/engine/reference/commandline/volume_inspect/)
+
+[5 - Docker Command Line Reference](https://docs.docker.com/engine/reference/commandline/volume_ls/)
+
+
+# Environment:
+This test requires access to VMware Nimbus cluster for dynamic ESXi, vCenter, and NFS server creation
+
+# Test Steps:
+1. Deploy VIC appliance to vCenter with a simple cluster and one NFS server; nfs share point added here.
```

Comment:
See comment above regarding install variations, but what happens when you try to use a read only share?  What happens when the share is nfsv4 only?
You don't need to implement all of these cases, but if they make sense as test cases then they should be documented at least and we can come back to add implementations (create a follow on story)